docs/monitors/mysql_monitor.md

```diff
@@ -74,6 +74,11 @@ You can also use this data in [Dashboards](/help/dashboards) and [Alerts](/help/
 |||# ``database_hostport``||| Hostname (or IP address) and port number of the MySQL server, e.g. ``dbserver:3306``, or \
                               simply ``3306`` when connecting to the local machine. You should specify one of \
                               ``database_socket`` or ``database_hostport``, but not both.
+|||# ``use_ssl``          ||| Whether or not to use SSL when connecting to the MySQL server. Defaults to False.
```

Comment:
Per test cases, if this config option is set, specified hostname will also be verified against the hostname in the cert, right?

Reply:
And minor thing - we should probably emit a warning (with a rate limit, e.g. couple of times per day) in case ``use_ssl`` is true, but ``ca_file`` is not specified.

IIRC, we already do the same thing in a couple of places.

Reply:
Only if `use_ssl` and also `ca_file` are specified will we verify the hostname. Just the `use_ssl` option alone does nothing except satisfy a mysql server configured with `require_secure_transport`. This is technically a valid use case but warning about it is probably smart.